### DISCUSSION TITLE
MobTrack: Track mobile remotely with a number

### DISCUSSION THREAD

**Discussion by SaikatGH75** [scaleracademy/open-source-projects#22] [Ideas]
**MobTrack: Track mobile remotely with a number** 

**Beta version**:
With this web / mobile application one can check the current location of a person. It can have the following features:
1.	While registering the designated mobile number in the application it will ask for some more information of the mobile user like below:
       a.	Mobile users home/residence location
       b.	Age of the mobile user (input can be the range)
       c.	Male or female 
       d.	Occupation

2.	With this application the user will not be able to get complete day’s journey history 

**Advanced version**:
In the future the application will provide the following features:
       a.	Get call log of the mobile user 
       b.	Get message log of the mobile user
       c.	Copy and forward important or subspecies messages to a registered mobile number or application

**Note**:
To get the benefit of the advanced features of ‘MobTrack’ the user needs to provide some more information of the mobile user like below:
      a.	IMEI number of the mobile
      b.	The mobile number is from which state

**Comment by Raj-saurabh** [comment]
Nice idea in terms of family member security purpose also.
